axum/Cargo.toml

```diff
@@ -40,7 +40,6 @@ serde = "1.0"
 serde_urlencoded = "0.7"
 sync_wrapper = "0.1.1"
 tokio = { version = "1", features = ["time"] }
-tokio-util = "0.6"
```

Comment:
Oh wow we're not using this anymore. Doesn't seem like it. Although please file a separate PR that removes this. A bit weird to have it mixed in with this.

Reply:
`tokio-util` only exists in `Cargo.toml`. It never used in `.rs` file.

Reply:
Yeah I get that. But please submit the removal in a separate PR. Its not related to ECOSYSTEM.md at.